src/cli.rs

```diff
@@ -259,28 +298,55 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                 let response = admin_client.start_admin(request).await?;
                 println!("{:?}", response.into_inner());
             }
+            
+            // senseicli --token <> listnodes 
             "listnodes" => {
-                let request = tonic::Request::new(ListNodesRequest { pagination: None });
+                let request = tonic::Request::new(ListNodesRequest { pagination: pagination.into() });
                 let response = admin_client.list_nodes(request).await?;
                 println!("{:?}", response.into_inner());
             }
+            
+            // senseicli --token <> createnode <username> <alias>
             "createnode" => {
                 let username = command_args.value_of("username").unwrap();
                 let alias = command_args.value_of("alias").unwrap();
 
+                let start: bool = command_args
+                    .value_of("start")
+                    .expect("start field required")
+                    .parse()
+                    .expect("start must be true or false");
+
+
                 let mut passphrase = String::new();
                 println!("set a passphrase: ");
                 io::stdin().read_line(&mut passphrase)?;
-
+                
                 let request = tonic::Request::new(CreateNodeRequest {
                     username: username.to_string(),
                     alias: alias.to_string(),
                     passphrase,
-                    start: false,
+                    start: start,
                 });
                 let response = admin_client.create_node(request).await?;
-                println!("{:?}", response.into_inner());
+                
+                let message = response.into_inner();
+                
+                let mut config = NodeConfig {
```

Comment:
After we get the node created back from senseid, save that data to the local (to the CLI) filesystem for future use.  I'm not exactly sure the best place to put this `data_dir` and would appreciate some input.